docs/snippets/solid/your-component-with-decorator.js.mdx

```diff
@@ -0,0 +1,21 @@
+```js
```

Comment:
If you could apply the same pattern here as to what was mentioned for the `storybook-preview-global-decorator.js.mdx` I'd appreciate it.

Reply:
Fixed.